docs/IE_PLUGIN_DG/Plugin.md

```diff
@@ -30,152 +30,141 @@ Based on that, declaration of a plugin class can look as follows:
 
-### `QueryNetwork()`
+### `query_model()`
 
 Use the method with the `HETERO` mode, which allows to distribute network execution between different 
 devices based on the `ov::Node::get_rt_info()` map, which can contain the `"affinity"` key.
-The `QueryNetwork` method analyzes operations of provided `network` and returns a list of supported
-operations via the InferenceEngine::QueryNetworkResult structure. The `QueryNetwork` firstly applies `TransformNetwork` passes to input `ov::Model` argument. After this, the transformed network in ideal case contains only operations are 1:1 mapped to kernels in computational backend. In this case, it's very easy to analyze which operations is supposed (`_backend` has a kernel for such operation or extensions for the operation is provided) and not supported (kernel is missed in `_backend`):
+The `query_model` method analyzes operations of provided `model` and returns a list of supported
+operations via the ov::SupportedOpsMap structure. The `query_model` firstly applies `transform_model` passes to input `ov::Model` argument. After this, the transformed network in ideal case contains only operations are 1:1 mapped to kernels in computational backend. In this case, it's very easy to analyze which operations is supposed (`m_backend` has a kernel for such operation or extensions for the operation is provided) and not supported (kernel is missed in `m_backend`):
 
 1. Store original names of all operations in input `ov::Model`
-2. Apply `TransformNetwork` passes. Note, the names of operations in a transformed network can be different and we need to restore the mapping in the steps below.
-3. Construct `supported` and `unsupported` maps which contains names of original operations. Note, that since the inference is performed using OpenVINO™ reference backend, the decision whether the operation is supported or not depends on whether the latest OpenVINO opset contains such operation.
-4. `QueryNetworkResult.supportedLayersMap` contains only operations which are fully supported by `_backend`.
+2. Apply `transform_model` passes. Note, the names of operations in a transformed network can be different and we need to restore the mapping in the steps below.
+3. Construct `supported` map which contains names of original operations. Note, that since the inference is performed using OpenVINO™ reference backend, the decision whether the operation is supported or not depends on whether the latest OpenVINO opset contains such operation.
+4. `ov.SupportedOpsMap` contains only operations which are fully supported by `m_backend`.
 
-@snippet template/src/plugin.cpp plugin:query_network
+@snippet template/src/plugin.cpp plugin:query_model
 
-### `SetConfig()`
+### `set_property()`
 
-Sets new values for plugin configuration keys:
+Sets new values for plugin property keys:
 
-@snippet template/src/plugin.cpp plugin:set_config
+@snippet template/src/plugin.cpp plugin:set_property
```

Comment:
it contains support of old metrics like SUPPORTED_METRICS and IMPORT_EXPORT_SUPPORT, can we drop it?

Reply:
why?